src/main/java/yuquiz/domain/study/service/StudyService.java

```diff
@@ -166,6 +178,41 @@ public void deleteUser(Long studyId, Long userId, Long deleteId) {
         studyUserRepository.deleteByStudy_IdAndUser_Id(studyId, deleteId);
     }
 
+    @Transactional(readOnly = true)
+    public Page<SeriesSummaryRes> getStudySeries(String keyword, Long studyId, Long userId, SeriesSortType sort, Integer page) {
+        if (!studyUserRepository.existsByStudy_IdAndUser_IdAndState(studyId, userId, UserState.REGISTERED)) {
+            throw new CustomException(StudyExceptionCode.UNAUTHORIZED_ACTION);
+        }
+
+        return seriesService.getStudySeriesSummary(keyword, studyId, sort, page);
+    }
+
+    @Transactional
+    public void createStudyPost(PostReq postReq, Long userId, Long studyId, boolean isNotice) {
+        if (isNotice) {
+            if (!validateLeader(studyId, userId)) {
+                throw new CustomException(StudyExceptionCode.UNAUTHORIZED_ACTION);
+            }
+        } else {
+            if (!studyUserRepository.existsByStudy_IdAndUser_IdAndState(studyId, userId, UserState.REGISTERED)) {
+                throw new CustomException(StudyExceptionCode.UNAUTHORIZED_ACTION);
+            }
+        }
```

Comment:
가독성이 조금 떨어지는 문장인 것 같습니다! `isNotice == true, !validateLeader == false`일 때 어떤 일이 일어날지 명확하게 보이지 않는 것 같아요. 아래와 같이 리팩토링 할 수 있을 것 같네요~

```suggestion
      boolean isAuthorized;
      
      if (isNotice) {
          isAuthorized = validateLeader(studyId, userId);
      } else {
          isAuthorized = studyUserRepository.existsByStudy_IdAndUser_IdAndState(studyId, userId, UserState.REGISTERED);
      }
      
      if (!isAuthorized) {
          throw new CustomException(StudyExceptionCode.UNAUTHORIZED_ACTION);
      }
```